### DISCUSSION TITLE
Custom 404 page?

### DISCUSSION THREAD

**Discussion by liamtsang** [honojs/hono#2252] [Q&A] [answered]
Is there a way to serve a custom 404 page whenever I throw a HTTPException(404)?
I know I can make a custom function to do this but am wondering if there is a way to always serve the same template whenever a 404 is thrown, specifically from inside pages with params as those are considered "existing" routes.

**Comment by yusukebe** [comment, marked as answer]
@liamtsang 

Use this method: https://hono.dev/api/hono#not-found

**Reply by liamtsang** [reply]
Ah thank you, I was searching "404" appreciate the help!
